docs/self-managed/modeler/web-modeler/configuration/configuration.md

```diff
@@ -68,6 +68,21 @@ Web Modeler integrates with Identity and Keycloak for authentication and authori
 
 Refer to the [advanced Identity configuration guide](./identity.md) for additional details on how to set up secure connections to an external Identity instance.
 
+### Zeebe Client
+
+Web Modeler uses the [Zeebe Java client](/docs/apis-tools/java-client/index.md) to connect to Zeebe.
+To customize the client configuration, you can provide optional environment variables.
+
+| Environment variable          | Description                                                                                              | Example value                    | Default value                |
+| ----------------------------- | -------------------------------------------------------------------------------------------------------- | -------------------------------- | ---------------------------- |
+| `ZEEBE_CA_CERTIFICATE_PATH`   | [optional]<br/>Path to a root CA certificate to be used instead of the certificate in the default store. | `/path/to/certificate`           | -                            |
+| `ZEEBE_CLIENT_CONFIG_PATH`    | [optional]<br/>Path to the client's OAuth credential cache.                                              | `/path/to/credentials/cache.txt` | `$HOME/.camunda/credentials` |
+| `ZEEBE_CLIENT_REQUESTTIMEOUT` | [optional]<br/>The request timeout used when communicating with a target Zeebe cluster.                  | `60000`                          | `10000`                      |
+| `ZEEBE_AUTH_CONNECT_TIMEOUT`  | [optional]<br/>The connection timeout for requests to the OAuth server.                                  | `30000`                          | `5000`                       |
+| `ZEEBE_AUTH_READ_TIMEOUT`     | [optional]<br/>The data read timeout for requests to the OAuth server.                                   | `30000`                          | `5000`                       |
+
+For more details, visit [Zeebe connection troubleshooting](/docs/self-managed/modeler/web-modeler/troubleshooting/troubleshoot-zeebe-connection.md).
```

Comment:
I think with these thinks you can remove the `/docs` and `/versioned_docs/version-8.3` sections.

Reply:
Done ✅